create_dev_meetings_report.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
Add the licence line at the top of the file

Reply:
the standard "# Licensed under a 3-clause BSD style license - see LICENSE.rst" I see in many other gammapy scripts?

Reply:
yes, exactly! :)